Update B2B-overview.adoc

mcherem opened the pull request:
Beta doc review - overview doc

mcherem committed:
Update B2B-overview.adoc

george-hoffer commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -27,6 +27,6 @@ FTP, FTPS, SFTP, JMS, SMTP, HL7 MLLP (only HL7 MLLP has its individual intro pag
 == Exchanges
 
-AS2, RNIF (each item has it’s individual intro page)
+AS2, RNIF (each item will have it’s individual intro page)
```
1. MuleSoft style discourages the use of "will" -- please use present voice.
2. "it's" is a typo in this context and should be "its" (possessive).
3. Better to use "introduction" instead of "intro", which is jargon.

mcherem replied:
This brings up a good point. The items in parenthesis are indications of what will come with the docs. So, this will be gone for Beta. I guess that I should not use the future tense even for indicating items.

mcherem committed:
Beta file review

Fileset for Beta review. Included are:
1. Partner Manager Setup for recieving message flows.
2. Alll other files required for X12 support for recieving message flows.

mcherem reviewed (commented):
Change per george.

mcherem committed:
Update B2B-overview.adoc

wwarner-salesforce committed:
Delete .DS_Store

wwarner-salesforce committed:
Delete .DS_Store

wwarner-salesforce committed:
Delete .DS_Store

wwarner-salesforce committed:
Delete .DS_Store

wwarner-salesforce committed:
Create .gitignore

mcherem committed:
Update nav.adoc for correct product structure.
